src/main/java/DAO/DBConnection.java

```diff
@@ -0,0 +1,34 @@
+package dao;
+
+import dao.exceptions.DAOException;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class DBConnection {
```

Comment:
If it`s singleton pattern we should use private constructor, otherwise other dev`s can create another objects from default constructor